### ISSUE TITLE
Return all "percent" metrics as a ratio, render as % in reports, plots as needed

### ISSUE THREAD

**Issue by rickyegeland** [ktindiana/sphinxval#75]
First discussed in SPHINX chat Feb 6, it was noted that metrics with "percent" in the label were not actually returned as a percentage, i.e. the result of a (amount/total) calculation was not multiplied by 100.  While the initial reaction was to do the multiplication by 100 for all metrics with percent in the name, uncertainty about the meaning of "percent" in log space and arguments about consistency of the range of outputs lead to extensive discussion in several group meetings.  Ultimately, the decision about whether to multiply or not comes down to convention, but by arguments of calculation and pseudo-consistency given below, **let it be decided to NOT multiply percent metrics by 100 in the metric computation code, nor to store these metrics with the multiplication by 100 in the SPHINX data outputs (e.g. csv, pkl)**.  The multiplication step is left to the code presenting the metric, if desired (e.g. in plotting code or reports).

Metrics with "percent" in the name:

"Mean Percent Error (MPE)": [],
"Mean Absolute Percent Error (MAPE)": [],
"Mean Percent Log Error (MPLE)": [],
"Mean Absolute Percent Log Error (MAPLE)": [],
"Mean Symmetric Percent Error (MSPE)": [],
"Mean Symmetric Absolute Percent Error (MSAPE)": [],
"Mean Symmetric Percent Log Error (MSPLE)": [],
"Mean Symmetric Absolute Percent Log Error (MSAPLE)": [],

### 1. We may legitimately use some of these numbers in other calculations

e.g. "Percent Correct", one could use this to estimate an expected number of correct forecasts, e.g. PC * N = N_correct.  If PC is stored in percent units, these calculations require a division by 100, which is awkward.

### 2. Most of our metrics lie in the range [-1, +1]

There is a small degree of consistency in avoiding a multiplication by 100 by keeping the range of the "percent" metrics similar to those with a range [0, 1] or [-1, +1].  Of course, the "error" metrics are technically unbounded, but nonetheless avoiding the rescaling helps with pseudo-consistency.

### 3. The percent sign is a unit; most of our metrics are unitless

The percent sign can be thought of as an (implied) attached unit (1 % = (1 / 100)), whereas many of our metrics are unitless ratios.  There is an increased degree of consistency in storing "percent" metrics without the unit.

### 4. Excel expects decimal numbers, multiplies by 100 to display them

Microsoft Excel is a widely used spreadsheet program that it could be argued sets many expectations in the ways data are handled in all of computing.

"For example, if a cell contains the number 10, Excel multiplies that number by 100, which means that you will see 1000.00% after you apply the Percentage format. This may not be what you expected. To accurately display percentages, before you format the numbers as a percentage, make sure that they have been calculated as percentages, and that they are displayed in decimal format. Percentages are calculated by using the equation amount / total = percentage."

https://support.microsoft.com/en-us/office/format-numbers-as-percentages-de49167b-d603-4450-bcaa-31fba6c7b6b4

### 5. Other python packages also ignore "percent" in a name

scipy percent point function (ppf()) takes its input significance level as a decimal value (e.g. 0.95, no 95 for the 95% significance level).  However, scipy percentiles and numpy.percentile do take integers [0, 100].

sklearn.metrics.mean_absolute_percentage_error is not returned as a percentage in the range [0, 100].  

torchmetrics mean_absolute_percentage_error MAPE and symmetric (SMAPE) also returns a ratio.

### Afterword

Ultimately, this issue is just about adopting a convention and sticking with it.  None of the above justifications indicate a large amount of work saved with respect to the opposite convention.

**Comment by ktindiana** [comment]
All metrics are in ratio in metrics.py, per a recent commit. 
post_analysis codes render in percent for plotting. 
Metrics that need to be converted to percent in post processing are:

in_percent = ["Mean Percent Error (MPE)",
                "Mean Absolute Percent Error (MAPE)",
                "Mean Symmetric Percent Error (MSPE)",
                "Mean Symmetric Absolute Percent Error (SMAPE)",
                "Median Symmetric Accuracy (MdSA)",
                "Mean Accuracy Ratio (MAR)"]

**Comment by rickyegeland** [comment]
@ktindiana to check if plots are handling this decision correctly.

**Comment by rickyegeland** [comment]
Looks like 681e185d handles the percentage representation in reporting, and done in post-analysis plots it was done before.  Closing.
